cli/smartcontract/smart_contract.go

```diff
@@ -818,9 +825,19 @@ func getAccFromContext(ctx *cli.Context) (*wallet.Account, *wallet.Wallet, error
 	var addr util.Uint160
 
 	wPath := ctx.String("wallet")
-	if len(wPath) == 0 {
+	walletConfigPath := ctx.String("wallet-config")
```

Comment:
...and generalize `getAccFromContext` that will be reused by both packages.

Reply:
The case is that each wallet-related command performs different actions with account. Sometimes we need to unlock it, sometimes not. Sometimes we're OK with default account. Sometimes it's OK to have several accounts. Also, accounts are hidden under different flag names. And all of these cases must be handled inside this single method.

Maybe we can reuse the code that opens wallet, but I'd prefer to keep `getAccFromContext` inside `smartcontract` package and do not generalize it.